monitor/external_export/graphana.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+import json
+import os
+import requests
+import string
+import argparse
+
+from monitor import get_lambda_names
+
+
+mon_home = os.getenv('DSS_MON_HOME')
+
+def load_template_file(template_path: str):
+    """Loads JSON template to dict."""
+    with open(template_path) as file:
+        data = json.load(file)
+    return data
+
+def format_panel_positioning(dashboard_src:dict, dashboard_dst: dict):
+    '''Use the dashboard_src to position panels correctly on the dashboard_dest'''
+    for panel in dashboard_dst['panels']:
+        for source_panel in dashboard_src.get("panels"):
+            if panel['title'] == source_panel['title']:
+                panel["gridPos"] = source_panel['gridPos']
+                break
+            print(f"Unable to locate source panel with title: {panel['title']}")
+        break
+    return dashboard_dst
+
+class DCPMetricsDash:
+    def __init__(self):
+        self.dcp_monitor_dashboard_url = 'https://raw.githubusercontent.com/HumanCellAtlas/dcp-monitoring/' \
+                                         'master/terraform/modules/env-dashboards/dss-dashboard.tf'
+        self.current_dashboard = self.get_current_dashboard()
+        self.unused_id = self.get_unused_panel_id()
+
+    def get_unused_panel_id(self):
+        ids = [ x for x in range(100) if x not in self.get_used_panel_ids() ]
```

Comment:
Hard-coding `100` is probably fine, but I would bump this a couple of zeros just in case.

Reply:
i dont expect the number of panels exceeding 100